ntbs-service/Models/Notification.cs

```diff
@@ -184,14 +185,34 @@ private string CreateNoAbodeOrPostcodeString()
             }
         }
 
+        private string CreateFormattedOccupationString()
+        {
+            if (PatientDetails?.Occupation == null)
+            {
+                return string.Empty;
+            }
+
+            if (PatientDetails.Occupation.HasFreeTextField && !string.IsNullOrEmpty(PatientDetails.OccupationOther))
+            {
+                return $"{PatientDetails.Occupation.Sector} - {PatientDetails.OccupationOther}";
+            }
+
+            if (PatientDetails.Occupation.Sector == "Other")
+            {
+                return PatientDetails.Occupation.Role;
+            }
+
+            return $"{PatientDetails.Occupation.Sector} - {PatientDetails.Occupation.Role}";
```

Comment:
Should we put all of this logic on the occupation model? then we still have access to it directly from Notification, but it all stays relevant to its fields? ie
`public string FormattedOccupationString => PatientDetails.Occupation.ToString();` up above? (not sure whether to actually override `ToString` or use a different name for the method)

Reply:
I definitely agree with this in principle, but for these purposes the method would have to be on the PatientDetails class (which contains OccupationOther field). Does this still merit the change for you?

Reply:
Ah, good point. Yeah, I think that's still worth it :)